catalog/nsili/catalog-nsili-endpoint/pom.xml

```diff
@@ -169,6 +169,12 @@
             <artifactId>catalog-email-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
+            <version>1.18</version>
+            <scope>test</scope>
```

Comment:
When running `mvn test`, this would fail because it couldn't find an apache `ArchiveException` included in this added dependency.